sr_moveit_hand_config/launch/scene.launch

```diff
@@ -0,0 +1,8 @@
+<launch>
```

Comment:
This launch file should be better placed inside the moveit config of the hand and arm rather that in the hand config only as it is meant to be used when both are present. In my opinion it should be placed here: https://github.com/shadow-robot/sr_interface/tree/kinetic-devel/sr_multi_moveit/sr_multi_moveit_config/launch